package.json

```diff
@@ -16,17 +16,18 @@
   "dependencies": {
     "@apollo/react-hooks": "3.1.3",
     "@apollo/react-ssr": "3.1.3",
-    "@auth0/nextjs-auth0": "^0.10.0",
+    "@auth0/nextjs-auth0": "0.10.0",
     "apollo-cache-inmemory": "1.6.5",
     "apollo-client": "2.6.8",
     "apollo-link-context": "1.0.19",
     "apollo-link-http": "1.5.16",
     "dotenv": "^8.2.0",
     "graphql": "14.6.0",
     "graphql-tag": "2.10.3",
-    "isomorphic-unfetch": "^3.0.0",
+    "isomorphic-unfetch": "3.0.0",
     "js-cookie": "2.2.1",
     "next": "9.3.2",
+    "next-absolute-url": "1.2.2",
```

Comment:
Should we keep this?

Reply:
That one's important for figuring out the domain from `req`... only one (canonical) domain is provided via environment variables, so you might have a user preview a PR at `nextjs-auth0-hasura-pull-request-branch-name.vercel.app`, while the VERCEL_URL is `nextjs-auth0-hasura-k14pehpuj.vercel.app`, and Auth0 wants the former domain in the callback URL during login.